README.md

```diff
@@ -62,6 +63,38 @@ druid's existing functionality and widgets.
 [![flex.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/flex.png)](./druid/examples/flex.rs)
 [![custom_widget.rs example](https://raw.githubusercontent.com/xi-editor/druid/screenshots/images/0.6.0/custom_widget.png)](./druid/examples/custom_widget.rs)
 
+## Goals
+
+Druid's primary goal is to provide a smooth and polished user experience with
+pixel perfect design and no behavioral quirks. In order to achieve this we
+strive for a variety of things:
+
+- Make it easy to build and package on all supported platforms.
+- Provide abstractions to avoid platform specific quirks.
+- Encourage good application behavior across platforms.
+- Automate platform resolution independence.
+- Enable robust internationalization.
+- Ensure accessibility to a wide group of people.
+- Produce small and fast binaries with low memory usage.
+- Have a small dependency tree with a high quality code base and good organization.
```

Comment:
I would say having a small dependency tree is an end goal. To put it differently, having a small dependency tree is one of the major reasons I'm personally interested in druid over Electron. I wrote in more detail about it #806.

Also the point about `cargo build` working is even more of a process goal, because that won't have any impact after the build step, while a small dependency tree will matter at runtime too.

Reply:
I guess I see "produce small and fast binaries with low memory usage" as being our shot at electron?

Reply:
Aside from being a good goal in general, I think it can be seen as such.